tools/proofers/for_mentors.php

```diff
@@ -254,12 +390,15 @@ function output_page_summary_table(Round $mentored_round, string $projectid)
 
 function output_page_list_table(Round $mentored_round, string $projectid)
 {
+    echo "<input type='hidden' name='projectid' value='$projectid'>";
     echo "<table class='striped basic'>";
     echo "<tr>";
     echo "<th>" . _("Page") . "</th>";
     echo "<th>" . _("Saved") . "</th>";
     echo "<th>" . _("Proofreader") . "</th>";
     echo "<th>" . _("WordCheck Events") . "</th>";
+    echo "<th>" . _("Current Page State") . "</th>";
+    echo "<th>" . _("Current Proofreader") . "</th>";
```

Comment:
> It's okay as is, but, would it be fair to truncate "WordCheck Events" to "WC Events" (not without approval from mentors, of course).

I'm fine with that, just need to loop back to the mentors.

> * "Current Page State" would be clearer if it said "Mentored Page State"
> * And if "Current Proofreader" simply said "Mentor".

Well, technically the software only knows what proofreader's name is in the column -- it doesn't know if they're actually a _mentor_. That might be splitting hairs and the difference unimportant but is what first crossed my mind.

> * And if it would be appropriate to have a time- and datestamp column for the "Which proofreader did each page."

You mean separate columns vs the joined "Saved" column already?

Reply:
> > * "Current Page State" would be clearer if it said "Mentored Page State"
> > * And if "Current Proofreader" simply said "Mentor".
> 
> Well, technically the software only knows what proofreader's name is in the column -- it doesn't know if they're actually a _mentor_. That might be splitting hairs and the difference unimportant but is what first crossed my mind.

True, and that's fair. My thinking process was that, for this script and this table, that column should only ever have mentors' names, but it's true that squirrels can see and under strict need proof pages.

> 
> > * And if it would be appropriate to have a time- and datestamp column for the "Which proofreader did each page."
> 
> You mean separate columns vs the joined "Saved" column already?

I didn't realize that it was joined. It's certainly not showing the P2 column datestamp, which is what I was looking for. As I said, though, that may be of interest only to me, and I don't feel very strongly about it.

Reply:
> It's certainly not showing the P2 column datestamp, which is what I was looking for.

Ah, so you're suggesting adding the P2 timestamp in addition to the P1 timestamp that's already there. Let me think about that.

Reply:
> > It's certainly not showing the P2 column datestamp, which is what I was looking for.
> 
> Ah, so you're suggesting adding the P2 timestamp in addition to the P1 timestamp that's already there. Let me think about that.

Yes, that's what I was looking for, but the more I think about it, the more I realize it's not really necessary from a mentoring perspective.

As I said to begin with, all three of the suggestions were possibly just things that made more sense from a testing perspective. I'm happy for them not to be implemented, and just mentioned them in case any might be worthwhile. I'm fine with going with what we have.

Reply:
Let's go with what we have here (I'd like to get @70ray's review on the LPage change when he's back which should be well before the Sept deploy). We can let the mentors know that we're open to more suggestions on how to improve this page as well.